src/trie/trie_type.c

```diff
@@ -88,7 +88,7 @@ Vector *Trie_Search(Trie *tree, char *s, size_t len, size_t num, int maxDist, in
     }
     // in prefix mode we also factor in the total length of the suffix
     if (prefixMode) {
-      ent->score /= sqrt(1 + fabs(slen - len));
```

Comment:
Whoops, I meant it's not safe to cast size_t to a signed type, since the standard doesn't specify what that type would be. And subtraction between unsigned types could underflow.